decredPoliteia/decredPoliteia_vo.md

```diff
@@ -3,29 +3,27 @@
 
 Named after the Greek term meaning ‘a system of government’--Politeia is the Decred proposal system: a chain-anchored social media application where stakeholders decide which projects recieve funding from the network fund. 
 
-Decred governance data hosted on Politeia is made up of submitted project proposals, comments, and votes. 
+Decred governance data hosted on Politeia is made up of submitted project proposals, comments, and up/down votes. 
```

Comment:
up/down votes would be awkward for the VO artist to say. Would probably be best expressed visually.